test_fms/mpp/test_mpp_init_logfile.F90

```diff
@@ -0,0 +1,44 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL Flexible Modeling System (FMS).
+!*
+!* FMS is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FMS is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+!***********************************************************************
+
+!> @file
+!! @brief Tests the mpp_init_logfile interface
+!! @author Miguel Zuniga
+!! @email gfdl.climate.model.info@noaa.gov
+
+program test_mpp_init_logfile
+#include <fms_platform.h>
```

Comment:
You should use platforms_mod instead

Reply:
Thats what I get for copy and pasting. I dont even use it.